src/opm/parser/eclipse/EclipseState/Schedule/Group/Group.cpp

```diff
@@ -109,9 +110,6 @@ const Group::GroupProductionProperties& Group::productionProperties() const {
     return this->production_properties;
 }
 
-const Group::GroupInjectionProperties& Group::injectionProperties() const {
-    return this->injection_properties;
-}
 
```

Comment:
The injectionProperties() is removed here and not from the .hpp file??? This method is used in line 1462 ParallelRestart.cpp.  Either it needs to be updated wrt multiple phases or the code in ParallellRestart needs to be changed.

Reply:
Yes - I know; I lost steam ...

Reply:
No problem. I had to hack a bit to get it to compile and work, but I now have a working code for a very simple example with multiple phases in group injection.  Do you plan to update this PR with the remaining parts? I can send a PR to your branch with sufficient changes to make it work.

Reply:
Thanks - I'll take your PR with a smile.